docs/tutorial-diffexp.rst

```diff
@@ -211,7 +211,7 @@ We can now check what we have created:
 We are now ready to run some expression and differential expression analyses.
 
 Cuffquant, Cuffnorm, and Cuffdiff
-================================
+=================================
 
```

Comment:
This should not be a separate commit. When not sure you can think of it as: "Does this deserve a CHANGELOG entry?".